dvc/stage/__init__.py

```diff
@@ -32,7 +32,7 @@
 logger = logging.getLogger(__name__)
 
 
-class Stage(schema.StageParams):
+class Stage(params.StageParams):
```

Comment:
This is also a pretty weird organisation :slightly_smiling_face: Though I understand that you did it to hack around quicker. We'll need to revisit all of this after and clean this up.